test/TestProjects/MgmtTypeSpec/src/Generated/Extensions/MgmtTypeSpecExtensions.cs

```diff
@@ -215,7 +215,7 @@ public static Pageable<MgmtTypeSpecPrivateLinkResourceData> GetAllPrivateLinkRes
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>PrivateLinks_Start</description>
+        /// <description>PrivateLinks_start</description>
```

Comment:
the original name is `start`: https://github.com/Azure/autorest.csharp/blob/371c2a2029ae8a2baef218b9b55798433f4b7b75/test/TestProjects/MgmtTypeSpec/PrivateEndpointConnection.tsp#L25

Reply:
Note that before `0.61.2`, it's still `Start`.